New in the Studio: Machine disaster recovery

mellson opened the pull request:
A new blog post about the new machine disaster recovery feature.

laurakalbag commented on content/posts/2022-12-13-new-in-the-studio-machine-disaster-recovery.mdx:
```diff
@@ -0,0 +1,46 @@
+
+At this moment in time, the Studio is using a traditional server-client architecture. Whenever you edit a machine, we will persist these changes to our central database, ready for you to use when you return - on any device with an internet connection.
+
+![Simplified drawing of the Stately Studio architecture.](./images/studio_architecture_challenges.png)
```
```suggestion
![Simplified drawing of the Stately Studio architecture described above.](./images/studio_architecture_challenges.png)
```

laurakalbag commented on content/posts/2022-12-13-new-in-the-studio-machine-disaster-recovery.mdx:
```diff
@@ -0,0 +1,46 @@
+
+![The Stately Studio shows an option to restore changes made to a machine while the user was offline.](./images/recovery_restore_machine.gif)
+
+Hopefully, you’ll never use this, but if you do, I hope you find this helpful.
```
```suggestion
Hopefully, you’ll never use our disaster recovery, but if you do, I hope you find this helpful.
```

laurakalbag commented on content/posts/2022-12-13-new-in-the-studio-machine-disaster-recovery.mdx:
```diff
@@ -0,0 +1,46 @@
+title: "New in the Studio: Machine disaster recovery"
+description: >-
+  We have introduced a new feature to the Stately Studio. A feature we hope
+  you’ll never see. Let’s call it disaster recovery, for lack of a better word.
```
```suggestion
  you’ll never see. Let’s call it disaster recovery, for lack of a better description.
```

laurakalbag reviewed (approved):
Yay, thank you, this is such a great post! I love your examples so much.

I switched the straight quotes to curly quotes, as is our Stately style. I’ve made a few tiny suggestions for clarity, but please feel free to discard.

Perhaps lines 30 + 32 and lines 36 + 38 could have the empty line between them removed? I think both of these sections read well as paragraphs, and they’re not too long when joined together.

It might aid understandability to add a couple of headings to break the content up and summarise for those who skim read, but as this post isn’t too long, you might think it unnecessary!

mellson commented:
> Yay, thank you, this is such a great post! I love your examples so much.
> 
> I switched the straight quotes to curly quotes, as is our Stately style. I’ve made a few tiny suggestions for clarity, but please feel free to discard.
> 
> Perhaps lines 30 + 32 and lines 36 + 38 could have the empty line between them removed? I think both of these sections read well as paragraphs, and they’re not too long when joined together.
> 
> It might aid understandability to add a couple of headings to break the content up and summarise for those who skim read, but as this post isn’t too long, you might think it unnecessary!

Thank you, I implemented your suggestions and also added the headers. Much better now. I'll release this once the feature is deployed.

davidkpiano commented on content/posts/2022-12-13-new-in-the-studio-machine-disaster-recovery.mdx:
```diff
@@ -0,0 +1,50 @@
+---
+title: "New in the Studio: Machine disaster recovery"
```
```suggestion
title: "New in the Studio: Machine recovery"
```

In general, wondering if we can make this sound a bit less scary 😅

mellson replied:
I agree; it could sound a little scary. I've turned it down a notch to a G Rated experience 😊
